hera_sim/visibilities/__init__.py

```diff
@@ -0,0 +1,34 @@
+"""
+A sub-package dedicated to defining various visibility simulators, where the
+focus is on simulating visibilities from a known sky-locked signal. These
+include simulators of varying levels of accuracy and performance capabilities,
+as well as varying expected input types -- eg. floating point sources,
+temperature fields, rectilinear co-ordinates, spherical co-ordinates, healpix
+maps etc. This package intends to unify the interfaces of these various kinds
+of simulators.
+"""
+from .simulators import VisibilitySimulator
+from .conversions import *
+
+import warnings
+
+# Registered Simulators
+from .vis_cpu import VisCPU
+
+
+try:
+    from .healvis_wrapper import HealVis
+except (ImportError, NameError) as e: # NameError arises bc healvis is not Python 3 compliant
+    warnings.warn("healvis failed to import.")
```

Comment:
Maybe a little nitpicky, but it would be good to include similar warnings if the other simulators fail to import. If you have the `warnings.warn(msg)` statement, then there's no need to `pass` as well. I also think that you can just do 
```
try:
    something
except [Exceptions,]:
    warnings.warn(msg)
```
rather than giving the exceptions an alias (without the `as e` part); the only time you really need to do that (as far as I know) is when you intend to do something with the exception.

Reply:
I've given them all a warning, and removed redundant `e`.